src/views/ConfigurationJoystickView.vue

```diff
@@ -321,11 +321,16 @@ import {
 import BaseConfigurationView from './BaseConfigurationView.vue'
 
 const controllerStore = useControllerStore()
-const { globalAddress } = useMainVehicleStore()
+const { globalAddress, isArmed, disarm } = useMainVehicleStore()
 
 const m2rSupportsExtendedManualControl = ref<boolean>()
 const ardupilotSupportsExtendedManualControl = ref<boolean>()
 onMounted(async () => {
+  // Automatically disarms the vehicle if it is armed and entering joystick configuration
+  if (isArmed) {
+    disarm()
+  }
+
```

Comment:
I think this auto-disarm we should actually remove. It's a too dangerous operation to be done automatically.

I think it's better for us to relegate that decision to the user than to simply disarm his vehicle without further notice.